code/game/machinery/portable_turret.dm

```diff
@@ -986,6 +986,11 @@
 		return 0
 	return 10
 
+/obj/machinery/porta_turret/syndicate/pod
+	health = 40
+	projectile = /obj/item/projectile/bullet/weakbullet3
+	eprojectile = /obj/item/projectile/bullet/weakbullet3

```

Comment:
In the test run it killed a bunch of confused people but once a couple of assistants figured out what was going on they just speared the turrets to death after pushing lockers for cover